src/backend/mod.rs

```diff
@@ -1,2 +1,26 @@
+//! Contains the different backends to get the screenshot from.
+
 pub mod wayland;
-pub mod xorg;
+pub mod x11;
+
+/// A general backend error enum.
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    /// Represents that an error occured while trying to get a screenshot on X11.
+    #[error(transparent)]
+    X11(#[from] x11::Error),
+}
+
+/// The main function of this module.
+///
+/// This function returns an rgb-image for each screen (or "monitor" in other
+/// words).
+pub fn get_images() -> Result<Vec<image::RgbImage>, Error> {
```

Comment:
We can't map the images to displays then can we?

Reply:
Also is image imported?

Reply:
> We can't map the images to displays then can we?

no, currently not. But this can be changed.

Reply:
> Also is image imported?

`image` refers to the crate [`image`](https://crates.io/crates/image) in this case ;)